src/codeflare_sdk/common/utils/unit_test_support.py

```diff
@@ -427,8 +428,18 @@ def create_cluster_all_config_params(mocker, cluster_name, is_appwrapper) -> Clu
         head_memory_requests=12,
         head_memory_limits=16,
         head_extended_resource_requests={"nvidia.com/gpu": 1, "intel.com/gpu": 2},
+        head_tolerations=[
+            V1Toleration(
```

Comment:
In terms of usage, I wonder if we should include the import `from kubernetes.client import V1Toleration` in the [codeflare-sdk __init__.py file](https://github.com/project-codeflare/codeflare-sdk/blob/main/src/codeflare_sdk/__init__.py)? - This is to avoid having to perform this import manually.

Reply:
I think it's a good idea too. Let's leave this as it is, but we should create a separate Jira for doc-input on the usage for users. - Which could then be used by Breda.